Sources/Kitura/CodableRouter.swift

```diff
@@ -408,6 +470,46 @@ extension Router {
         }
     }
 
+    // Get w/Optional Query Parameters
+    fileprivate func getSafely<Q: QueryParams, O: Codable>(_ route: String, handler: @escaping (Q?, @escaping CodableArrayResultClosure<O>) -> Void) {
+        get(route) { request, response, next in
+            Log.verbose("Received GET (plural) type-safe request with Query Parameters")
+            Log.verbose("Query Parameters: \(request.queryParameters)")
+            do {
+                var query: Q? = nil
+                let queryParameters = request.queryParameters
+                if queryParameters.count > 0 {
+                  query = try QueryDecoder(dictionary: request.queryParameters).decode(Q.self)
```

Comment:
Indentation is off.

Reply:
Also, you can just do `dictionary: queryParameters`

Reply:
done